nym-vpn-core/Cargo.toml

```diff
@@ -172,27 +172,27 @@ zeroize = "1.6.0"
 
 # 3d2914b3e5d707cd56e45508cfda6b656d483190 points to the current develop (as of 08.01.25) that contains
```

Comment:
remove this comment because it was referring to the pinned commit

Reply:
Thanks for pointing this out. Removed the obsolete comment.